doc/examples.rst

```diff
@@ -10,3 +10,10 @@ Examples
     examples/visualization_gallery
     examples/ROMS_ocean_model
     examples/ERA5-GRIB-example
+
+apply_ufunc
+------------
+.. toctree::
+    :maxdepth: 2
+
```

Comment:
nbsphinx automatically adds sections from the notebooks, so unless you want to add more `apply_ufunc` examples you might want to remove the explicit section to be consistent with the other notebooks.

Reply:
>  unless you want to add more apply_ufunc examples

Yes, this is my intent.

Reply:
Then, would it be better to replace `apply_ufunc` with "Using `apply_ufunc`" or something like that?

Reply:
:+1: